src/main/kotlin/uk/gov/justice/digital/hmpps/pecs/jpc/controller/HtmlController.kt

```diff
@@ -76,16 +82,23 @@ class HtmlController(@Autowired val moveService: MoveService, @Autowired val jou
     fun journeys(
             @ModelAttribute(name = DATE_ATTRIBUTE) @DateTimeFormat(iso = DateTimeFormat.ISO.DATE) startOfMonth: LocalDate,
             @ModelAttribute(name = SUPPLIER_ATTRIBUTE) supplier: Supplier,
-            @ModelAttribute(name = "locationName") locationName: String,
-            @ModelAttribute(name = "agencyId") agencyId: String,
+            @ModelAttribute(name = "flashAttrMappedLocationName") locationName: String?,
+            @ModelAttribute(name = "flashAttrMappedAgencyId") agencyId: String?,
             model: ModelMap,
     ): String {
 
+        removeAttributesIf(locationName.isNullOrEmpty(), model, "flashAttrMappedLocationName", "flashAttrMappedAgencyId")
```

Comment:
The feels wrong but without this Thymeleaf was rendering text the first time you go into the journeys page when we don't want it.  It should only show the text if a location has been mapped.